### ISSUE TITLE
NVDA not reading PowerPoint Outline view (just reads linefeed)

### ISSUE THREAD

**Issue by SMTestGH** [nvaccess/nvda#12719]
### Steps to reproduce:
1. Enable NVDA 2021.1
2. Open PowerPoint
3. Select blank presentation
4. Type text into the Title text box on slide 1
5. Navigate to View > Outline view Alt>W>PO
6. Press F6 until focus lands in Outline view
7. Arrow up/down to read text

### Actual behavior:
NVDA doesn't read the text in the Outline view. Only reads "linefeed" and "blank"

### Expected behavior:
NVDA is able to read all text in the Outline view

#### NVDA installed/portable/running from source:
Portable

#### NVDA version:
2021.1

#### Windows version:
21H1 version 19043

#### Name and version of other software in use when reproducing the issue:
PowerPoint for Office 365 - any build from 2019 and up

#### Other information about your system:
Issue does not occur using Narrator - text read correctly. Likely presented via UIA that NVDA is not capturing.

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Repros in old versions too (2020)

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by KD38223** [comment]
I verify this issue is still occurring. All parameters outlined above are the same for my testing. Support or a fix would be appreciated.
